packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts

```diff
@@ -96,6 +100,7 @@ export const defaultLivenessDisplayText: Required<LivenessDisplayText> = {
   hintIlluminationTooDarkText: 'Move to brighter area',
   hintIlluminationNormalText: 'Lighting conditions normal',
   hintHoldFaceForFreshnessText: 'Hold still',
+  hintMatchIndicatorA11yText: 'face fit.',
```

Comment:
See above comment on line 16:
```suggestion
  hintMatchIndicatorText: 'face fit.',
```